### ISSUE TITLE
Cache bust static resources

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#1100]
**TL;DR**: We should implement some way of letting the browser know that the static files have changed.  

**Narrative**
When we update files, browser will hold last file in cache and attempt to use that file.  This diminishes the user experience just after an update. 

There are two solutions to this issue:
1. Implement code that will add a hash to the static file names.  
2. Implement a HTTP caching scheme that will allow files to be readily re-downloaded.

**Implementation Notes**
Here are some resources.  
- https://simonhearne.com/2022/caching-header-best-practices/
- https://docs.djangoproject.com/en/3.2/ref/contrib/staticfiles/#django.contrib.staticfiles.storage.ManifestStaticFilesStorage
- https://stackoverflow.com/a/5321976

**Comment by becky-gilbert** [comment]
We attempted to fix this here: #1538 using file names with hashes, but it didn't work (caused 500 errors on the site). We need to find a way to reproduce the remote static file build/serve/settings locally in order to debug.

**Comment by okaycj** [comment]
With the complex storage of this site and the unnecessary nature of this feature.  We should probably focus on more important issues.  Closing.
